frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx

```diff
@@ -192,9 +204,32 @@ export const VMDisksFirehose: React.FC<VMLikeEntityTabProps> = ({ obj: vmLikeEnt
     }),
   ];
 
+  const vm =
+    kindObj === VirtualMachineModel && isVM(vmLikeEntity)
```

Comment:
I see that this is how we do it in other places too ... maybe in a follow up - IMHO we can make a helper method to extract vm and vmi from the props, and use in all the places we have this code ... ?

Reply:
Agreed!